modDbImpl/src/org/aion/db/impl/leveldb/LevelDB.java

```diff
@@ -293,7 +294,9 @@ public boolean isEmpty() {
         check();
 
         try {
-            return new LevelDBIteratorWrapper(db.iterator());
+            ReadOptions readOptions = new ReadOptions();
```

Comment:
can you move this object to a member variable? I think it can get a little performance/memory gain

Reply:
looks like it can't be avoided to use the object here. just ignore my comment.